### ISSUE TITLE
🐞 Cannot import name 'JsonConverter'

### ISSUE THREAD

**Issue by patrikbraborec** [dagger/dagger#3790]
### What is the issue?

Hi, I am having trouble with Python SDK. I went through the documentation and wanted to execute the script from docs:

```python
"""
Execute a command
"""

import sys
import anyio
import dagger

async def test():
    config = dagger.Config(log_output=sys.stderr)

    async with dagger.Connection(config) as client:

        python = (
            client.container()

            # pull container
            .from_("python:3.10-slim-buster")

            # get Python version
            .exec(["python", "-V"])
        )

        # execute
        version = await python.stdout().contents()

        print(f"Hello from Dagger and {version}")

if __name__ == "__main__":
    anyio.run(test)
```

And got the following output:

### Log output

```bash
Traceback (most recent call last):
  File "/Users/patrikbraborec/_PROJECTS/gooddata/open-source/good-ci-cd/test.py", line 7, in <module>
    import dagger
  File "/Users/patrikbraborec/_PROJECTS/gooddata/open-source/good-ci-cd/venv/lib/python3.10/site-packages/dagger/__init__.py", line 5, in <module>
    from .server import Server
  File "/Users/patrikbraborec/_PROJECTS/gooddata/open-source/good-ci-cd/venv/lib/python3.10/site-packages/dagger/server/__init__.py", line 6, in <module>
    from cattrs.preconf.json import JsonConverter
ImportError: cannot import name 'JsonConverter' from 'cattrs.preconf.json' (/Users/patrikbraborec/_PROJECTS/gooddata/open-source/good-ci-cd/venv/lib/python3.10/site-packages/cattrs/preconf/json.py)
```

### Dagger version

dagger 0.2.8 (92c8c7a2) darwin/amd64

### OS version

MacOS 13.0 (22A380), Python 3.10.8

**Comment by helderco** [comment]
May be a new breaking version of `cattrs`, let me investigate.

**Comment by helderco** [comment]
Can't replicate, see if you have latest version of `cattrs`.

https://github.com/python-attrs/cattrs/blob/405f0291b958ae9eb45ee38febeb91fb65dd644f/src/cattrs/preconf/json.py#L14

**Comment by patrikbraborec** [comment]
Sorry, I missed your reply. I will try it during the day. Thanks!

**Comment by patrikbraborec** [comment]
Fixed by installing` pip install cattrs==22.2.0`. I had 22.1.0 and it caused the mentioned problem. Thanks for your reply!

**Comment by helderco** [comment]
It's a dependency of `dagger-io`, we actually have minimum `22.2.0`, so no need to install it manually just update.

https://github.com/dagger/dagger/blob/a8035349a43363c3c7fbd296f0f3711f14803040/sdk/python/pyproject.toml#L45
